src/products/Product.java

```diff
@@ -1,102 +1,94 @@
 package products;
 
-import enums.KindOfProduct;
```

Comment:
Per kind of product userei enum, ma solo per quello, dal momento che la consegna parla di un gestionale in grado di gestire questi tre tipi di prodotti. Così si può anche fare la ricerca per tipo di prodotto